roles/openshift_node_group/tasks/sync.yml

```diff
@@ -70,5 +70,36 @@
     - __status_of_sync_ds.results.results[0].status.desiredNumberScheduled is defined
     - __status_of_sync_ds.results.results[0].status.numberAvailable == __status_of_sync_ds.results.results[0].status.desiredNumberScheduled
   retries: 60
-  delay: 30
-  failed_when: false
+  delay: 10
+
+- name: Wait for sync DS to set annotations on all nodes
+  oc_obj:
+    state: list
+    kind: node
+    selector: ""
+  register: node_status
+  until:
+    - node_status.results is defined
+    - node_status.results.results is defined
+    - node_status.results.results | length > 0
+    - node_status.results.results[0]['items']
+     | map(attribute='metadata.annotations') | map('list') | flatten
```

Comment:
Ansible 2.4 doesn't have `flatten` filter

Reply:
Nice, I assume you found this out manually testing?

Reply:
Yes, this needs more fixes, I'll open a new PR instead